src/utility/razor_imu_9dof/nodes/imu_node.py

```diff
@@ -225,12 +225,28 @@ def reconfig_callback(config, level):
     words = string.split(line,",")    # Fields split
     if len(words) > 2:
         #in AHRS firmware z axis points down, in ROS z axis points up (see REP 103)
+        '''NOTE: Make changes to the coordinate system here
+        '''
         yaw_deg = -float(words[0])
-        yaw_deg = yaw_deg + imu_yaw_calibration
+        #yaw_deg = yaw_deg + imu_yaw_calibration
+        yaw_deg = yaw_deg + mag_declination
```

Comment:
Please provide comment noting static configuration, and possibly difference from [line 231](https://github.com/SJSU-AD/FusionAD/pull/117/files#diff-25ae3a2587c604a98fb12fe5298e54b4R231), for future reference, in case re-configuration is ever necessary for debugging

Reply:
Removed changes to driver.